mass_mailing_custom_unsubscribe/static/src/js/unsubscribe.js

```diff
@@ -0,0 +1,274 @@
+/* Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+/* This JS module replaces core AJAX submission because it is impossible
+ * to extend it as it is currently designed. It is almost a copy+paste from
+ * upstream, to allow easier version/patch updates, so linter is disabled
+ * and prettier is ignore. */
```

Comment:
@Tardo I'll mark as resolved some of your comments because of what is explained here (btw @pedrobaeza you can see the comment is already in place).